epoch/api/README.md

```diff
@@ -62,6 +63,33 @@ A response (and an event) has the format:
 | action | string | what was the action | Yes |
 | payload | object | data from the action | Yes |
 
+## Channels WebSocket API definition
+
+### Description
+Channels require persisted connections to an Aeternity node. New state is
+co-signed by both parties and then it becomes the latest valid state of the
+channel. This happens off chain. For persistence of the connection websockets
+are used.
+
+### Connection
+The epoch node supports an endpoint with a configurable port where the
```

Comment:
Please rebase on master that has improvement in endpoint organization https://github.com/aeternity/protocol/commit/b45a3d04f32f310dd65ab4c9190a7db9185b5056